src/map/utils/attackutils.cpp

```diff
@@ -294,25 +294,28 @@ namespace attackutils
         float occ_extra_dmg        = battleutils::GetScaledItemModifier(PChar, PWeapon, Mod::OCC_DO_EXTRA_DMG) / 100.f;
         int16 occ_extra_dmg_chance = battleutils::GetScaledItemModifier(PChar, PWeapon, Mod::EXTRA_DMG_CHANCE) / 10;
 
-        if (occ_extra_dmg > 3.f && occ_extra_dmg_chance > 0 && xirand::GetRandomNumber(100) <= occ_extra_dmg_chance)
+        if (allowRelicProc)
```

Comment:
we should use a different name for this bool, as its entirely plausible for some other item to someday have the same behavior and be controller in the same ,manner in the same spot.

Reply:
> we should use a different name for this bool, as its entirely plausible for some other item to someday have the same behavior and be controller in the same ,manner in the same spot.

I changed the bool to `allowRelicOccDamageProc`, does that work? Or do you mean something else?

Reply:
I was leaning toward just not having the word "relic" in there, make it about what it does rather than what item(s) its for.

Reply:
ow do you feel about just allowProc or even allowOccDmgProc ? my point had been we might someday have something not a relic in that block.